include/gridtools/storage/adapter/nanobind_adapter.hpp

```diff
@@ -24,76 +24,86 @@
 
 namespace gridtools {
     namespace nanobind_sid_adapter_impl_ {
+#if NB_VERSION_MAJOR >= 2
+    using _ssize_t = nanobind::ssize_t;
+#else
+    using _ssize_t = std::size_t;
```

Comment:
I’d prefer another name here, e.g., `stride_t` or so. Calling an unsigned type similar to the signed `ssize_t` looks a bit misleading to me. Further, apart from the `_impl` namespace and some old `gcl` code, we don’t seem to use names starting with underscore.